resources/analysisTools/indelCallingWorkflow/checkSampleSwap_TiN.pl

```diff
@@ -216,17 +229,16 @@
 
 close GTraw;
 
-## Annotating with dbSNP
-
-my $runAnnotation = system("cat $snvsGT_RawFile | perl $ANNOTATE_VCF -a - -b '$gnomAD' --columnName='gnomAD_COMMON_SNV' --reportMatchType  --bAdditionalColumn=2 | perl $ANNOTATE_VCF -a - -b '$localControl' --columnName='LocalControl_COMMON_SNV' --reportMatchType --bAdditionalColumn=2 | perl $ANNOTATE_VCF -a - -b '$localControl_2' --columnName='LocalControl2_COMMON_SNV' --reportMatchType --bAdditionalColumn=2 > $snvsGT_gnomADFile");
+## Annotating with gnomAD and local control 
+my $runAnnotation = system("cat $snvsGT_RawFile | perl $ANNOTATE_VCF -a - -b '$gnomAD' --columnName='gnomAD_GENOMES' --bAdditionalColumn=2 --reportOnlyMatches --reportMatchType --reportLevel 4 | perl $ANNOTATE_VCF -a - -b '$localControl' --columnName='LocalControl_2018' --bAdditionalColumn=2  --reportOnlyMatches --reportMatchType --reportLevel 4 > $snvsGT_gnomADFile");
```

Comment:
Consider quoting the filenames. This command will be interpreted by the shell where the value of `$nsvsGT_gnomADFile` will be interpreted as file. If it contains special characters or spaces, the interpretation will be incorrect. Therefore, just for safety reasons, quoting is advisable:

```R
system("cat '$snvsGT_RawFile' | perl '$ANNOTATE_VCF' -a - -b '$gnomAD' --columnName='gnomAD_GENOMES' --bAdditionalColumn=2 --reportOnlyMatches --reportMatchType --reportLevel 4 | perl '$ANNOTATE_VCF' -a - -b '$localControl' --columnName='LocalControl_2018' --bAdditionalColumn=2  --reportOnlyMatches --reportMatchType --reportLevel 4 > '$snvsGT_gnomADFile'");
```

Reply:
@vinjana These files are created by the pipeline, so I assume no one working with the pipeline will add 'space' to the file names in the future, but sure we can 'future-proof' the code.